### PR TITLE
Include prerequisites in the releasing guide.

### PR DESCRIPTION
**Pull request by swankjesse**
https://github.com/cashapp/barber/blob/master/docs/releasing.md

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+
+   ```
+   $ gpg --keyring secring.gpg --export-secret-key XXXXXXXX > ~/.gnupg/secring.gpg
+   ```
+
+
+Cutting a Release
+-----------------
+
 1. Update `CHANGELOG.md`.
```
Comment (monkey-mas) [inline docs/releasing.md]: just an idea: is it useful if it's hyperlinked? it doesn't have to be though :)
```
[CHANGELOG.md][changelog_url]

[changelog_url]: https://github.com/square/okhttp/blob/master/CHANGELOG.md
```

File: docs/releasing.md [hunk 2]
```diff
@@ -1,6 +1,63 @@
...
+
+Upload the GPG keys to public servers:
+
+```
+$ gpg --list-keys --keyid-format LONG
+/Users/johnbarber/.gnupg/pubring.kbx
+------------------------------
+pub   rsa4096/XXXXXXXXXXXXXXXX 2019-07-16 [SC] [expires: 2029-07-13]
+      YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY
+uid           [ultimate] John Barber <jbarber@squareup.com>
```
Comment (monkey-mas) [inline docs/releasing.md]: lol

File: docs/releasing.md [hunk 3]
```diff
@@ -10,7 +67,14 @@ Releasing
     export NEXT_VERSION=X.Y.Z-SNAPSHOT
```
Comment (monkey-mas) [inline docs/releasing.md]: just wanted to make sure. is the example of this like below? 👀 
```
export RELEASE_VERSION=4.0.2
export NEXT_VERSION=4.0.3-SNAPSHOT
```

Comment (swankjesse) [reply docs/releasing.md]: yep!

**Pull request merged**